src/applications/simple-forms/form-upload/containers/FormUploadApp.jsx

```diff
@@ -5,7 +5,7 @@ import PropTypes from 'prop-types';
 
 import { RequiredLoginView } from '@department-of-veterans-affairs/platform-user/RequiredLoginView';
 
-function FormUploadApp({ user }) {
+const FormUploadApp = ({ user }) => {
   useEffect(() => {
     document.title = 'Form Upload | Veterans Affairs';
   }, []);
```

Comment:
While you're refactoring, I think you can get rid of the `connect` logic all together via `useSelector`:

```js
import React, { useEffect } from 'react';
import { useSelector } from 'react-redux'; 
import { Outlet } from 'react-router-dom-v5-compat';
import PropTypes from 'prop-types';

import { RequiredLoginView } from '@department-of-veterans-affairs/platform-user/RequiredLoginView';

const FormUploadApp = ({ user }) => {
  const user = useSelector(state => state.user);
   
  useEffect(() => {
    document.title = 'Form Upload | Veterans Affairs';
  }, []);

  return (
    <RequiredLoginView user={user} serviceRequired={[]}>
      <Outlet />
    </RequiredLoginView>
  );
};

FormUploadApp.propTypes = {
  user: PropTypes.object,
};

export default FormUploadApp;
```